feature-pack/src/main/resources/modules/system/layers/base/jakarta/security/enterprise/api/main/module.xml

```diff
@@ -22,13 +22,13 @@
   ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
   -->
 
-<module xmlns="urn:jboss:module:1.5" name="javax.security.enterprise.api">
+<module xmlns="urn:jboss:module:1.5" name="jakarta.security.enterprise.api">
```

Comment:
Please leave the module name (and therefore the file name) unchanged.

Reply:
Fixed.